packages/create-ponder/src/index.ts

```diff
@@ -201,6 +202,17 @@ export const run = async (
     prettier.format(tsConfig, { parser: "json" })
   );
 
+  const eslintConfig = `
+    {
+      "extends": "eslint-config-ponder"
```

Comment:
Pretty sure this can just be `"extends": "ponder"` 🤓

Reply:
yes!